include/s/s_Math.h

```diff
@@ -22,6 +22,14 @@ void addCalcScaled(float *value, float stepSize, float maxStep);
 int absDiff(short a1, short a2);
 BOOL chaseUC(u8 *value, u8 target, u8 stepSize);
 
+template <typename T>
```

Comment:
This function already exists in c_math. If this is a better place for it we should remove the other one.

Reply:
Understood. nsmbw has it named in sLib